doc/source/using-ray-with-tensorflow.rst

```diff
@@ -320,3 +320,38 @@ For reference, the full code is below:
     # and 0.3 used in generate_fake_x_y_data.
     if iteration % 20 == 0:
       print("Iteration {}: weights are {}".format(iteration, weights))
+
+.. autoclass:: ray.experimental.TensorFlowVariables
+   :members:
+
+Troubleshooting
+---------------
+
+Note that ``TensorFlowVariables`` uses variable names to determine what variables get set when calling ``set_weights``.
+One common issue when sharing weights between networks is that the setter and getter networks are both defined
```

Comment:
One common issue arises when two networks are defined in the same TensorFlow graph. In this case, TensorFlow appends an underscore and integer to the names of variables to disambiguate them.